modules/seqexec/server/src/main/scala/seqexec/server/gcal/GcalKeywordReader.scala

```diff
@@ -18,13 +18,13 @@ trait GcalKeywordReader {
 
 object DummyGcalKeywordsReader extends GcalKeywordReader {
 
-  def getDiffuser: SeqAction[Option[String]] = None
+  def getDiffuser: SeqAction[Option[String]] = None.toSeqActionO
```

Comment:
Oh was this an implicit conversion before? 👍 for explicit syntax.

Reply:
yes, it was implicit and very confusing for me